composer/loggers/console_logger.py

```diff
@@ -134,6 +130,9 @@ def predict_start(self, state: State, logger: Logger) -> None:
             self._log_hparams_to_console()
 
     def eval_start(self, state: State, logger: Logger) -> None:
+        self.logged_metrics = {}
+        # Clear logged metrics before eval so they don't get logged
+        # subsequently.
```

Comment:
nit: let's one line as much as possible
```suggestion
        self.logged_metrics = {}  # Clear logged metrics before eval so they don't get logged
```